### PR TITLE
~ | start of the pong game in godot

### PR DESCRIPTION
**Pull request by BolvicBolvicovic**
The game launches but is not working as expected.

### CODE REVIEW

**Comment by demostanis** [comment]
git commit -m '+ | Start the pong game in Godot' -m 'The games launches but isn't working as expected'

File: pong/scripts/game.gd [hunk 1]
```diff
@@ -0,0 +1,53 @@
+extends Node2D
+
+var screen_size
+var pad_size
+var direction = Vector2(1.0, 0.0)
+const INITIAL_BALL_SPEED = 80
```
Comment (demostanis) [inline pong/scripts/game.gd]: please separate consts and vars

File: pong/scripts/game.gd [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+	get_node("right").position = right_pos
+
+func _ready():
+	screen_size = get_viewport_rect().size
+	pad_size = get_node("left").get_texture().get_size()
+	set_process(true)
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+#func _process(delta):
```
Comment (demostanis) [inline pong/scripts/game.gd]: remove that maybe?

**Comment by demostanis** [comment]
please don't commit .mono, it contains .dlls...

**Comment by demostanis** [comment]
also we should add gdformat to .lintstagedrc

**Comment by demostanis** [comment]
> also we should add gdformat to .lintstagedrc

#4

**Review by DinoMalin — COMMENTED** [review]
The game isn't great yet, but we are getting somewhere

**Review by DinoMalin — APPROVED** [review]
Little physic problem when you try to speed the ball

**Pull request closed without merging**